database/src/main/kotlin/no/nav/su/se/bakover/database/behandling/BehandlingPostgresRepo.kt

```diff
@@ -115,4 +116,20 @@ internal class BehandlingPostgresRepo(
         }
         return hentBehandling(behandling.id)!!
     }
+
+    override fun finnesBehandlingForSøknad(søknadId: UUID): Boolean {
```

Comment:
Kunne kanskje skrevet denne i retning av noe slikt?

`return dataSource.withSession { session ->
            "select * from behandling where søknadId=:soknadId".hentListe(
                mapOf("soknadId" to søknadId), session
            ) {
                it.stringOrNull("søknadId")
            }
        }.isNotEmpty()`

Reply:
👍